platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -79,11 +80,11 @@ describe('OHIF Cornerstone Toolbar', () => {
 
     //drags the mouse inside the viewport to be able to interact with series
     cy.get('@viewport')
-      .trigger('mousedown', 'top', { which: 1 })
-      .trigger('mousemove', 'center', { which: 1 })
+      .trigger('mousedown', 'center', { which: 1 })
```

Comment:
changed click from `center` to `top` because it seems sometimes the click was not working properly. Clicking first in the center is more assertive.